docs/tutorials/super-for-jq.md

```diff
@@ -579,20 +600,34 @@ To appreciate all this, let's have a look next at some real-world data...
 
 ## Real-world GitHub Data
 
-Now that we've covered the basics of `zq` and the Zed language, let's
+Now that we've covered the basics of `super` and its query language, let's
 use the query patterns from above to explore some GitHub data.
 
 First, we need to grab the data.  You can use `curl` for this or you can
-just use `zq` as `zq` can take URLs in addition to file name arguments.
+just use `super` as it can take URLs in addition to file name arguments.
 This command will grab descriptions of first 30 PRs created in the
-public `zed` repository and place it in a file called `prs.json`:
+public `super` repository and place it in a file called `prs.json`:
 ```
 super -f json \
   https://api.github.com/repos/brimdata/super/pulls\?state\=all\&sort\=desc\&per_page=30 \
   > prs.json
 ```
+
+{{% tip "Note" %}}
+
+As we get into the exercise below, we'll reach a step where we encounter some
+unexpected empty objects in the original data. It seems the GitHub API
+must have been having a bad day when we first ran this exercise, as these
+empty records no longer appear if the download is repeated today using the same
+URL shown above. But taming glitchy data is a big part of data discovery, so to
+relive the magic of our original experience, you can download
+[this archived copy](https://superdb.org/docs/tutorials/prs.json) of the
+`prs.json` we originally saw.
+
+{{% /tip %}}
```

Comment:
It's always bothered me a little that the steps in the doc were not strictly repeatable if the reader followed our guidance about downloading the GitHub data anew. Rather than risk confusion, I figured we might as well just disclose.